lib/rubocop/cop/util.rb

```diff
@@ -14,10 +14,10 @@ module Util
 
       # http://phrogz.net/programmingruby/language.html#table_18.4
       # Backtick is added last just to help editors parse this code.
-      OPERATOR_METHODS = %w(
-        | ^ & <=> == === =~ > >= < <= << >>
-        + - * / % ** ~ +@ -@ [] []= ! != !~
-      ).map(&:to_sym) + [:'`']
+
+      OPERATOR_METHODS = [
+        :|, :^, :&, :<=>, :==, :===, :=~, :>, :>=, :<, :<=, :<<, :>>, :+,
+        :-, :*, :/, :%, :**, :~, :+@, :-@, :[], :[]=, :!, :!=, :!~, :`]
```

Comment:
This was a similar change as before. This file was failing for module length too long. This seemed like an easy place to save a line.